src/components/ColorPicker/tests/ColorPicker-ssr.test.tsx

```diff
@@ -19,11 +18,15 @@ jest.mock('../../../utilities/target', () => ({
 
 describe('<ColorPicker /> Server-side only', () => {
   it('does not attach the touchmove handler to the window', () => {
-    const colorPicker = mountWithAppProvider(
+    const colorPicker = mountWithApp(
       <ColorPicker color={red} onChange={noop} />,
     );
 
-    colorPicker.find(Slidable).first().simulate('mousedown');
+    colorPicker!.find(Slidable)!.find('div')!.trigger('onMouseDown', {
```

Comment:
colorPicker shall always be defined and will never be null so you don't need the first  `!` to manually assert that.

```suggestion
    colorPicker.find(Slidable)!.find('div')!.trigger('onMouseDown', {
```